src/Parsers/Gm1File.cpp

```diff
@@ -148,8 +148,8 @@ bool Gm1File::LoadFromDisk(ghc::filesystem::path path, bool cached) {
 
   /* Read compressed images into buffer */
   uint32_t buflen = Parser::GetLength() - offData;
```

Comment:
size_of_images

Reply:
compressed textures ?

Reply:
How about `n_textures`? It’s the number of textures, not their size.

Reply:
You are reading raw bytes here.

Reply:
See line 163

Reply:
Ah, ok.